File: Javascript/primes.js
function isPrime(n) {
    if(n < 2){
      return false
    }
    if (n % 2 === 0 && n > 2) {
      return false
    }else{
      for (let i = 2; i < n; i++) {
        if( n % i === 0){
          return false
        }
      }
    }
    return true
}

console.log(isPrime(2))
console.log(isPrime(73))
console.log(isPrime(10007))
console.log(isPrime(10002))
  

File: Python/primes.py
import math

def isPrime(n):
    if(n == 1 or n == 0):
        return False
    else:
        for i in range(2, int(math.sqrt(n))):
            if( n % i == 0):
                return False
        return True



def main():
    
    n = int(input("Enter a number: "))
    
    if isPrime(n):
        print(str(n) + " is prime")
    else:
        print(str(n) + " is not prime")
  

if __name__ == "__main__":
    main()
